### PR TITLE
Remove the deprecated "files" option for external grading

### LINKED ISSUE #1588: Officially remove support for externalGradingOptions.files
Per a `grep` on production as of the time of this issue, only 6 questions were still using `externalGradingOptions.files` in a question's `info.json`:

```
$ grep -r --include="info.json" "\"files\":" /data1/courses/
./pl-cs498sl/questions/fibonacciEditor/info.json:        "files": ["python_autograder/"],
./pl-cs498sl/questions/fibonacciUpload/info.json:        "files": ["python_autograder/"],
./pl-cs241/questions/dev_0_fa17/info.json:        "files": ["ocaml-grader/"],
./pl-cs418/questions/coding_test/info.json:        "files": ["javascript_autograder/"],
./pl-template/questions/fibonacciEditor/info.json:        "files": ["python_autograder/"],
./pl-template/questions/fibonacciUpload/info.json:        "files": ["python_autograder/"],
```

This option has been deprecated for [close to two years](https://github.com/PrairieLearn/PrairieLearn/commit/14816daa5a7268d60c5e3e7bac333614a96c207c); it's time to finally remove support. We can just manually push changes to these courses; once that's done, we can remove the extra code from `sync/fromDisk/questions.js` and `schemas/schemas/infoQuestion.json`.

### PR DESCRIPTION
**Pull request by nwalters512**
Removes remaining references to `externalGradingOptions.files` from question `info.json` files.

Closes #1588

### CODE REVIEW

**Comment by mwest1066** [comment]
Remind me exactly which course repos will need changing because of this? Are you thinking to do Shephard PRs for them?

**Comment by mwest1066** [comment]
Now I just read #1588 to answer all my questions 😄 

So manual PRs? File them already before we merge this?

File: ChangeLog.md [hunk 1]
```diff
@@ -278,6 +278,8 @@
   * Remove `string_from_2darray_sf()` from `freeformPythonLib/prairielearn.py` (Liz Livingston)
 
   * Remove `number` column from `question_tags` table; question tags are now sorted by `tags.number` (Nathan Walters).
+
+  * Remove support for `externalGradingOptions.files` in question `info.json` files (Nathan Walters).
```
Comment (mwest1066) [inline ChangeLog.md]: Maybe say `support for the deprecated XXX option`, so we don’t sound totally irresponsible? 😄

**Comment by nwalters512** [comment]
@mwest1066 the changes were trivial and non-breaking, so I just manually made the changes and pushed them to each course.

**Pull request merged**